src/org/labkey/test/pages/study/specimen/ManageNotificationsPage.java

```diff
@@ -0,0 +1,175 @@
+import org.labkey.test.Locator;
+import org.labkey.test.WebDriverWrapper;
+import org.labkey.test.WebTestHelper;
+import org.labkey.test.components.html.Checkbox;
+import org.labkey.test.components.html.Input;
+import org.labkey.test.components.html.RadioButton;
+import org.labkey.test.pages.LabKeyPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+import java.util.List;
+
+public class ManageNotificationsPage extends LabKeyPage<ManageNotificationsPage.ElementCache>
+{
+    public ManageNotificationsPage(WebDriver driver)
+    {
+        super(driver);
+    }
+
+    public static ManageNotificationsPage beginAt(WebDriverWrapper webDriverWrapper)
+    {
+        return beginAt(webDriverWrapper, webDriverWrapper.getCurrentContainerPath());
+    }
+
+    public static ManageNotificationsPage beginAt(WebDriverWrapper webDriverWrapper, String containerPath)
+    {
+        webDriverWrapper.beginAt(WebTestHelper.buildURL("study-samples", containerPath, "manageNotifications"));
+        return new ManageNotificationsPage(webDriverWrapper.getDriver());
+    }
+
+    public ManageNotificationsPage setReplyTo(@Nullable String email)
+    {
+        if (email == null)
+        {
+            elementCache().replyToAdminRadio.check();
+        }
+        else
+        {
+            elementCache().replyToUserRadio.check();
+            elementCache().replyToUserInput.set(email);
+        }
+        return this;
+    }
+
+    public ManageNotificationsPage setSubjectSuffix(String suffix)
+    {
+        elementCache().subjectSuffixInput.set(suffix);
+        return this;
+    }
+
+    public ManageNotificationsPage setNotificationUsers(@Nullable List<String> users)
+    {
+        if (users == null)
+        {
+            elementCache().newRequestNotifyCheckbox.uncheck();
+        }
+        else
+        {
+            elementCache().newRequestNotifyCheckbox.check();
+            shortWait().until(ExpectedConditions.visibilityOf(elementCache().newRequestNotifyInput.getComponentElement()));
+            elementCache().newRequestNotifyInput.set(String.join("\n", users));
+        }
+        return this;
+    }
+
+    public ManageNotificationsPage setCcUsers(@Nullable List<String> users)
+    {
+        if (users == null)
+        {
+            elementCache().ccCheckbox.uncheck();
+        }
+        else
+        {
+            elementCache().ccCheckbox.check();
+            shortWait().until(ExpectedConditions.visibilityOf(elementCache().ccInput.getComponentElement()));
+            elementCache().ccInput.set(String.join("\n", users));
+        }
+        return this;
+    }
+
+    public ManageNotificationsPage setDefaultEmailNotification(DefaultEmailNotify option)
+    {
+        scrollIntoView(elementCache().saveButton); // Make sure radio buttons are fully in view
+        elementCache().defaultEmailNotifyRadio(option).check();
```

Comment:
Why not scroll the option into view? I'm assuming that the button is scrolled into view because it is next to the options.